### DISCUSSION TITLE
<title>Category in Podcast Generator 3,2,9

### DISCUSSION THREAD

**Discussion by emmebrus** [community/community#42941] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, I'm new here and my english isn't so good, sorry. I have Podcast generator in my web space. When I upload two Episodes, one can be seen in the right Category, but the other no, the Category seems to be empty.
www.radiocooperativa.org/3/potcastgen is the link. Can someone help me? Thank you very much. emmebrus

**Comment by OwaisAzad** [comment]
It looks like there may be an issue with the way the podcast episodes are being categorized on your website. It's possible that the episodes are not being assigned to the correct category, or that there is a problem with the way the categories are being displayed.
